wolfcrypt/src/port/Renesas/renesas_sce_util.c

```diff
@@ -220,11 +223,11 @@ static int SCE_ServerKeyExVerify(uint32_t type, WOLFSSL* ssl, const uint8_t* sig
 
         if (ret != FSP_SUCCESS) {
             WOLFSSL_MSG("failed R_SCE_TLS_ServerKeyExchangeVerify");
-            cbInfo->pk_key_set = 0;
+            cbInfo->flags1.bits.pk_key_set = 0;
```

Comment:
Looks like `flags1` should be updated across this file, now that the name has been changed.

Reply:
Sure. Let me compile them on Windows again.

Reply:
Verified compile with the changes.